package.json

```diff
@@ -14,15 +14,13 @@
     "t7e": "0.2.3",
     "underscore": "1.4.2",
     "zooniverse": "zooniverse/Zooniverse#98ffd4b",
-    "zooniverse-geordi-client": "^1.0.0",
-    "zooniverse-user-string-getter": "^1.1.1",
+    "zooniverse-geordi-client": "^1.3.6",
+    "zooniverse-user-string-getter": "^1.1.22",
     "zootorial": "0.2.4"
   },
   "scripts": {
     "start": "hem server --port 6371",
-    "deploy-to-staging": "hem build && publisssh ./public zooniverse-demo/serengeti && rm -rf ./public/application.*",
+    "deploy-to-staging": "hem build && publisssh ./public zooniverse-static/preview.zooniverse.org/serengeti && rm -rf ./public/application.*",
     "deploy-to-production": "hem build && publisssh ./public zooniverse-static/www.snapshotserengeti.org && rm -rf ./public/application.*",
-    "deploy-experiment": "hem build && publisssh ./public zooniverse-static/www.snapshotserengeti.org/happyuserexperiment && rm -rf ./public/application.*",
```

Comment:
removing temporary/experimental scripts and old demo deploy script (since we now use preview instead)